docs/migration/cosmoshub-3.md

```diff
@@ -227,6 +253,92 @@ Cross check your genesis hash with other peers (other validators) in the chat ro
     Automated audits of the genesis state can take 30-120 min using the crisis module. This can be disabled by 
     `gaiad start --x-crisis-skip-assert-invariants`.
 
+# Guidance for Full Node Operators
+
+1. Verify you are currently running the correct version (v2.0.15) of _gaiad_:
+
+   ```bash
+    $ gaiad version --long
+    name: gaia
+    server_name: gaiad
+    client_name: gaiacli
+    version: 2.0.15
+    commit: 89cf7e6fc166eaabf47ad2755c443d455feda02e
+    build_tags: netgo,ledger
+    go: go version go1.15 darwin/amd64
+   ```
+
+1. Make sure your chain halts at the right time and date:
```

Comment:
I don't think full node operators should set halt time.

Basically only validator nodes need to.